sftpplus-docker-setup.sh

```diff
@@ -31,7 +33,7 @@ ln -s sftpplus-${SFTPPLUS_OS}-x64-${SFTPPLUS_VERSION} sftpplus
 # Initialize SFTPPlus configuration.
 # This will generate the SSH keys and the self signed certificates.
 cd sftpplus
-./bin/admin-commands.sh initialize
+./bin/admin-commands.sh initialize --init-admin admin --init-password pass
```

Comment:
I don't know if for docker usage, customers will want to initialized SFTPPlus for each image.

The old initialize command was there only to create a new self signed cert and ssh keys. 

I think that we should keep the server.ini file, and instead use explicit command to create the SSH keys like

```
./bin/admin-commands.sh generate-ssh-key --key-skip --key-type rsa --key-size 2048 --key-file /path/to/rsa_host
```

now we don't have a command to generate a self-signed key.

I will introduce it in v4.3.0 and we can use it here.

Reply:
Well, this repo is for evaluation purposes, so I'm fine with how things stand.

Of course, the above would be a nice addition, to be documented here as well. But I would still keep this repo self-sufficient, not requiring extra stuff to quickly fire up a container to try.

Reply:
Came back to this now that 4.3.0 is released, but I can't find any `server.ini` file in the trial packages (I've tried the `rhel8` one). What do you mean by "we should keep the server.ini file"? Thanks